unity-renderer/Assets/DCLServices/ScreencaptureCamera/Service/ScreencaptureCameraService.cs

```diff
@@ -50,7 +53,7 @@ public void Dispose()
                 Object.Destroy(enableCameraButtonCanvas.gameObject);
         }
 
-        public async UniTask InitializeAsync(CancellationToken cancellationToken)
+        private async UniTask InitializeAsyncInternal(CancellationToken cancellationToken)
```

Comment:
I think it's would be better to put the `Async` world at the end of the methods. Wdyt?
```suggestion
        private async UniTask InitializeInternalAsync(CancellationToken cancellationToken)
```

Reply:
good point, especially because we have it like that in our code conventions